client/react/hooks.tsx

```diff
@@ -485,7 +485,7 @@ function _usePasswordless() {
         tokensCb: (tokens) => storeTokens(tokens).then(() => setTokens(tokens)),
       });
       signinIn.signedIn.catch(setLastError);
-      return signinIn.signedIn;
+      return signinIn;
```

Comment:
Why this change? you added then the `.signedIn` on all other parts of code calling these functions, didn't you?

Reply:
I saw it wasn't consistent, other "signInWith..." methods returned the `signingIn` object, which holds not just the `signedIn` promise but also the `abort` function which may be useful to the caller––so I wanted to make it always available in the response. So, made it consistent and return the `signingIn` object always. If that makes sense?